
Epoch 0: 100%|██████████| 48/48 [00:00<00:00, 78.34it/s, v_num=iqiy, train_loss_step=0.173, val_loss_step=0.0677, val_loss_epoch=0.0711, train_loss_epoch=0.0688]
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
  | Name      | Type     | Params
---------------------------------------
0 | model     | RNNModel | 2.3 K
1 | criterion | MSELoss  | 0
---------------------------------------
2.3 K     Trainable params
0         Non-trainable params
2.3 K     Total params
0.009     Total estimated model params size (MB)
c:\Users\janfr\anaconda3\envs\pytorch_3_10\lib\site-packages\pytorch_lightning\trainer\connectors\data_connector.py:441: The 'val_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=15` in the `DataLoader` to improve performance.
c:\Users\janfr\anaconda3\envs\pytorch_3_10\lib\site-packages\pytorch_lightning\trainer\connectors\data_connector.py:441: The 'train_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=15` in the `DataLoader` to improve performance.





























Epoch 29: 100%|██████████| 48/48 [00:01<00:00, 41.58it/s, v_num=iqiy, train_loss_step=0.0599, val_loss_step=0.0662, val_loss_epoch=0.073, train_loss_epoch=0.0633]
Testing DataLoader 0: 100%|██████████| 6/6 [00:00<00:00, 274.40it/s]364
364
363
0
`Trainer.fit` stopped: `max_epochs=30` reached.
Restoring states from the checkpoint path at .\RNN_single_step_forecasts\ate6iqiy\checkpoints\epoch=29-step=1440.ckpt
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Loaded model weights from the checkpoint at .\RNN_single_step_forecasts\ate6iqiy\checkpoints\epoch=29-step=1440.ckpt
